scripts/install-deps.sh

```diff
@@ -72,7 +72,7 @@ install_githooks=false
 
 du_test_data_dir_path="/tmp/adu/"
 # DO Deps
-default_do_ref=v1.0.0
```

Comment:
for develop dont we want to lock to do's develop branch?

also, this doesn't fix our release which is still broken for ubuntu 20 by default.